### DISCUSSION TITLE
Help with arping and profiles.

### DISCUSSION THREAD

**Discussion by remay** [rsmarples/dhcpcd#82] [Q&A] [answered]
Hi,

I'm struggling with a configuration that I thought would be easy, but I'm making a meal of it.

I'd like to give an interface a static configuration unless it's connected to my development environment in which case I'd like to to use dhcp.

Simplifying:
```
interface ethN
arping 192.168.1.1
static ip_address=192.168.77.1/24

# In the dev environment the interface for 192.168.1.1 has mac address aa:bb:cc:dd:ee:ff
profile aa:bb:cc:dd:ee:ff
static ip_address=192.168.1.10/24
```
appears to work and on my dev environment, and gives the interface 192.168.1.10.  The interface gets 192.168.77.1 elsewhere.  Is there a way to get the dev environment to use dhcp.  I tried both an empty profile and a a profile with only a `dhcp` option, but neither appeared to work.

Thanks for any help,
Rob.

**Comment by rsmarples** [comment, marked as answer]
Hi, sorry for the late reply.

dhcpcd currently does not suppot this! Oddly enough the reverse of your situation is what most people want.
Anyway, I've added this commit to fix this here: b3c41d25a3c715a3d605950fd97bee9c6f99584f

So this config now works:
```
interface ethN
arping 192.168.1.1
static ip_address=192.168.77.1/24

# In the dev environment the interface for 192.168.1.1 has mac address aa:bb:cc:dd:ee:ff
profile aa:bb:cc:dd:ee:ff
static ip_address=
```
